### ISSUE TITLE
Crashes with WebStorm 2019.2 EAP

### ISSUE THREAD

**Issue by koresar** [jimkyndemeyer/js-graphql-intellij-plugin#259]
## Version and Environment Details
__Operation system__: OSX

__IDE name and version__: WebStorm 2019.2 EAP, but works fine with 2019.1 (stable one)

__Plugin version__: 2.1.0

## Actual Behaviour

```
Access is allowed from event dispatch thread only.

com.intellij.openapi.diagnostic.RuntimeExceptionWithAttachments: EventQueue.isDispatchThread()=false Toolkit.getEventQueue()=com.intellij.ide.IdeEventQueue@7cdec35e
Current thread: Thread[ApplicationImpl pooled thread 7,4,Idea Thread Group] 1111340881
SystemEventQueueThread: Thread[AWT-EventQueue-0 2019.2#WS-192.4787.13 WebStorm, eap:true, os:Mac OS X 10.14.4, java-version:JetBrains s.r.o 11.0.3+12-b281.38 2019.2#WS-192.4787.13 WebStorm, eap:true, os:Mac OS X 10.14.4, java-version:JetBrains s.r.o 11.0.3+12-b281.38,6,Idea Thread Group] 1110601899
	at com.intellij.openapi.application.impl.ApplicationImpl.assertIsDispatchThread(ApplicationImpl.java:1076)
	at com.intellij.openapi.application.impl.ApplicationImpl.assertIsDispatchThread(ApplicationImpl.java:1065)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.getEditors(FileEditorManagerImpl.java:1357)
	at com.intellij.lang.jsgraphql.v1.ide.project.JSGraphQLLanguageUIProjectService.lambda$reloadEndpoints$3(JSGraphQLLanguageUIProjectService.java:222)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:910)
	at com.intellij.lang.jsgraphql.v1.ide.project.JSGraphQLLanguageUIProjectService.reloadEndpoints(JSGraphQLLanguageUIProjectService.java:221)
	at com.intellij.lang.jsgraphql.v1.ide.project.JSGraphQLLanguageUIProjectService.onEndpointsChanged(JSGraphQLLanguageUIProjectService.java:204)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.intellij.util.messages.impl.MessageBusConnectionImpl.deliverMessage(MessageBusConnectionImpl.java:102)
	at com.intellij.util.messages.impl.MessageBusImpl.doPumpMessages(MessageBusImpl.java:440)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpWaitingBuses(MessageBusImpl.java:401)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:390)
	at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:374)
	at com.intellij.util.messages.impl.MessageBusImpl.sendMessage(MessageBusImpl.java:367)
	at com.intellij.util.messages.impl.MessageBusImpl.lambda$createTopicHandler$1(MessageBusImpl.java:237)
	at com.sun.proxy.$Proxy115.onEndpointsChanged(Unknown Source)
	at com.intellij.lang.jsgraphql.ide.project.graphqlconfig.GraphQLConfigManager.doBuildConfigurationModel(GraphQLConfigManager.java:657)
	at com.intellij.lang.jsgraphql.ide.project.graphqlconfig.GraphQLConfigManager.lambda$null$6(GraphQLConfigManager.java:498)
	at com.intellij.lang.jsgraphql.ide.project.graphqlconfig.GraphQLConfigManager$5.lambda$run$0(GraphQLConfigManager.java:516)
	at com.intellij.openapi.project.DumbService.lambda$runReadActionInSmartMode$1(DumbService.java:153)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:944)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:57)
	at com.intellij.openapi.project.DumbService.runReadActionInSmartMode(DumbService.java:146)
	at com.intellij.lang.jsgraphql.ide.project.graphqlconfig.GraphQLConfigManager$5.run(GraphQLConfigManager.java:513)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:893)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:171)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:593)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:539)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:158)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:410)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:299)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

## Steps to Reproduce / Link to Repo with Reproduction and Instructions

Just open the IDE

**Comment by jimkyndemeyer** [comment]
Thanks for catching this.

This relates to #256 and is caused by new assertions that JetBrains introduced. I'm currently looking at how to work around this, but no solution has presented itself.

**Comment by jimkyndemeyer** [comment]
Fixed in the 2.1.1 release, pending JetBrains approval.
